src/components/project/project.rules.ts

```diff
@@ -804,12 +806,21 @@ export class ProjectRules {
             'project_closing@tsco.org',
           ],
         };
-      default:
+
+      case ProjectStep.DidNotDevelop:
+      case ProjectStep.Rejected:
         return {
-          approvers: [Role.Administrator],
+          approvers: [
+            Role.Administrator,
+            Role.ProjectManager,
+            Role.RegionalDirector,
+            Role.FieldOperationsDirector,
+          ],
           transitions: [],
+          getNotifiers: async () => [...(await this.getProjectTeamUserIds(id))],
```

Comment:
I don't think the approvers does anything here as there are no transitions anyways. Also the notifiers could be simplified.
```suggestion
          approvers: [],
          transitions: [],
          getNotifiers: () => this.getProjectTeamUserIds(id),
```

Reply:
ok, I wasn't sure who could approve to these statuses so I pinged Seth to get his feedback, but will revise per your feedback.